app/script/view_model/content/InputBarViewModel.js

```diff
@@ -489,15 +505,15 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
   }
 
   handleMentionFlow() {
-    const textarea = document.querySelector('#conversation-input-bar-text');
+    const textarea = this.getTextArea();
     const {selectionStart, selectionEnd, value} = textarea;
```

Comment:
I suppose we no longer need the const then and can simply do:

`const {selectionStart, selectionEnd, value} = this.getTextArea();`